plrust/Cargo.toml

```diff
@@ -28,6 +28,7 @@ target_postgrestd = []
 cfg-if = "1" # platform conditional helper
 once_cell = "1.7.2" # polyfills a nightly feature
 semver = "1.0.14"
+memfd = "0.6.2" # for loading user function .so
```

Comment:
Does this need to be inside `[target.'cfg(target_os="linux")'.dependencies]`?

Reply:
It doesn’t need to but I suppose it should. Will fix